### PR TITLE
Ser 227 nutrition training

### PR DESCRIPTION
**Pull request by Bahatiroben**
Adding nutrition training to the trainings menu

test from here https://telerivet.com/p/0c6396c9/service/SV7684c662f118f82e/edit
- dial in the kenya shortcode
- add account number 26215997
- choose 4 on the menu for trainings
- choose 0 to go to the next screen (2)
- choose option 12 for nutrition trainings on screen 2
- The 2 way sms service should be triggered.(at the moment the two way SMS is not live yet. it is in another PR to be reviewed next)
-There will also be a notice telling that a training has been initiated

to review the 2 way sms, test from here https://telerivet.com/p/0c6396c9/service/SV4a9437d2100a778c
 more focused tests will be demoed on a real phone with a Kenyan sim

### CODE REVIEW

File: nutrition-training/translations/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    'sms_2.1': {
+        'en-ke': 'You can get eggs, beans and vegetables from your farm and save money! Plant beans in small quantities and rear chicken for eggs.',
+        'sw': '(5/6)Unaweza kupata mayai, maharagwe na mboga kutoka shamba lako na kuokoa pesa! Panda maharagwe na ufuge kuku wa mayai. '
+    },
+    'sms_2.2': {
+        'en-ke': 'Contragulations! You have completed the training on dietary diversity! Practice what you have learnt in this lesson for a healthy family',
+        'sw': '(6/6)Hongera! Umemaliza mafunzo ya lishe bora! ZIngatia masomo haya kwa familia yako ili wawe na afya nzuri!'
+    },
+    'invalid_choice': {
+        'en-ke': 'Reply with a or b to answer',
```
Comment (shejasabin) [inline nutrition-training/translations/index.js:67]: it looks like the options are more than a and b in the above translations

Comment (Bahatiroben) [reply nutrition-training/translations/index.js:67]: Ohh missed that. It was like that in the translation file, and I forgot to add another option when I copied. Just adding it.

**Review by shejasabin — DISMISSED** [review]
I think you will have to deal with conflicts for option 11 on the menu though

File: nutrition-training/translations/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        'b. Ugali\n' +
+        'c. Mchele\n' +
+        'Jibu na  a, b, au c'
+    },
+    'sms_1.3': {
+        'en-ke': 'Correct! Beans, eggs, omena and green leafy vegetables (Managu, Saga) are very good sources of iron. Iron makes your child smart!',
+        'sw': 'Sahihi! maharagwe, mayai, omena na mboga za majani (managu) vina madini ya kuongeza damu. Madini haya hufanya mtoto wako kuwa mwerevu!'
+    },
+    'sms_1.4': {
+        'en-ke': 'Actually, beans have the most iron. Ugali/Rice make you full but do not contain enough iron to prevent anemia and imrove health',
```
Comment (nyabongo) [inline nutrition-training/translations/index.js:27]: improve is misspelt here

**Review by nyabongo — CHANGES_REQUESTED** [review]

File: nutrition-training/translations/index.js (hunk 2 above)
Comment (Bahatiroben) [reply nutrition-training/translations/index.js]: > improve is misspelt here

updated.

**Review by DevFojo — DISMISSED** [review]

**Review by nyabongo — COMMENTED** [review]
Since this does not actually send the 2 way sms I think it is incomplete to be in testing. can you target this branch with the PR that completes it, then we can merge this branch into develop after the next branch is merged into this one.

**Comment by Bahatiroben** [comment]
> Since this does not actually send the 2 way sms I think it is incomplete to be in testing. can you target this branch with the PR that completes it, then we can merge this branch into develop after the next branch is merged into this one.

Ok sounds good. done.

File: nutrition-training/translations/index.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+module.exports = {
+    'message_notification': {
+        'en-ke': 'A training SMS has been sent to your phone. Messages are free! But please delete unwanted SMS in your phone to make space for incoming.',
+        'sw': 'Ujumbe wa mafunzo umetumwa kwa simu yako. SMS zote ni bure! Kumbuka kufuta SMS usizohitaji ili uwe na nafasi ya kupata ujumbe zaidi. '
+    },
+    'sms_1.1': {
+        'en-ke': '(1/6)Today we will learn about blanaced diets. Eating like this prevents anemia and improves the energy and health of the whole family!',
```
Comment (nyabongo) [inline nutrition-training/translations/index.js:7]: typo in "... learn about **blanaced** diets"

**Review by nyabongo — CHANGES_REQUESTED** [review]

File: nutrition-training/translations/index.js (hunk 3 above)
Comment (Bahatiroben) [reply nutrition-training/translations/index.js]: @nyabongo updated.

**Review by DevFojo — APPROVED** [review]

**Review by nyabongo — APPROVED** [review]

**Review by DevFojo — APPROVED** [review]

**Pull request merged**